mopidy_spotify/lookup.py

```diff
@@ -92,3 +95,30 @@ def _lookup_playlist(config, session, web_client, uri):
     if playlist is None:
         raise spotify.Error("Playlist Web API lookup failed")
     return playlist.tracks
+
+
+def _lookup_your(config, session, web_client, uri):
+    parts = uri.replace("spotify:your:", "").split(":")
+    if len(parts) != 1:
+        return
+    variant = parts[0]
+
+    items = browse._load_your_music(web_client, variant)
+    if variant == "tracks":
```

Comment:
One thing different between the two varients here is that tracks won't be loaded by libspotify whereas albums will. I wonder if the difference is observable

Reply:
Would you be able to elaborate here? I'm pretty sure I tested this quite extensively at the time I wrote it. Unless something has changed with some other component, it should work.

Reply:
When you load any sort of container thing in libspotify it loads the track data for that in the background so if you then go and use the track (I.e. play it) it should happen instantly. If the tracks are being loaded via the Web API, when you go to play it that'll be the first time libspotify hears about the track and first has to load the data. There might be a short pause for that but I wasn't sure it'd be noticeable.

Reply:
Is there an alternative that would avoid that potential pause, and would it be worth scrapping what I've worked on in this PR to implement it vs merging this now and implementing that later?

Reply:
The only way I know (and it's a little hacky) is to do something like [this](https://github.com/mopidy/mopidy-spotify/blob/master/mopidy_spotify/playlists.py#L98-L110). I'll test it myself later but if you didn't notice any issue it's probably fine. I'd be happy to merge as it is and we can always come back later if people find the performance is an actual problem.

Reply:
I did wonder what the purpose of that code was.

Either way, I definitely think this feature is better off existing in a non-perfect state than not existing at all. I'll make the translator-related change you suggested to the lookup code at some point over the next couple of days, and then we can get this merged :+1: